blog_post.md

```diff
@@ -0,0 +1,13 @@
+Welcome to another edition of Methods You Never Knew You Needed!  We here at SHLM Tech greatly appreciated all of the feedback we received following last week's post on the "merchants_that_only_sold_one_item_but_somehow_had_two_items_returned" method.  We can't believe how many of you wonderful readers reached out to let us know that this was the method you were missing in your lives.  
+
+This week we are coming at you with two hot new methods for the multi-merchant sales platform.  The first one, "most_sold_item_for_merchant" allows for a merchant id to be used to search for a merchant's most sold item.  First it creates an array of the given merchant's items using the "find_all_by_merchant_id method.  It iterates through these items and creates an array of item ids.  It then finds the invoice items that are associated with these item ids and creates an array of the ones that were successful.  Next it uses these invoice items to create a hash called "total_quantity" with item ids as keys and the number of units sold as their corresponding values.  These values are then put into an array and the method .max is used to find the item with the highest quantity.   
```

Comment:
I dont know how much they are going to read into this but the keys in the has are the item instances and not the item ids!

Reply:
I love the style that this is written in, nice job @lgsriclas 😂 

It might be helpful to either add in the methods themselves for easy comparison and/or use an ordered list to detail the steps that the method takes in addition to the paragraph explanation.

Reply:
I love the humor.  Nice job.  I think putting in the methods themselves might it too long.  Are we just doing one of the methods, or am I missing something?